public/app/plugins/datasource/influxdb/response_parser.ts

```diff
@@ -2,7 +2,7 @@ import _ from 'lodash';
 
 export default class ResponseParser {
   parse(query: string, results: { results: any }) {
-    if (!results || results.results.length === 0) {
+    if (!results?.results || results.results.length === 0) {
```

Comment:
Nit: Could also be

```suggestion
    if (!results?.results?.length === 0) {
```

Reply:
🚀🚀🚀

Reply:
I had to revert this, cause it can't be  `(!results?.results?.length === 0)` as if the `length === 0`, then it is true, but we have `!` so it would become false. And the other way around, if the `length !== 0` then it if false and `!` makes it ture, which we don't want.